src/components/Sum.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Sum = ({ a = 0, b = 0 }) => (
```

Comment:
follow Ihor's advice regarding named exports
```suggestion
export const Sum = ({ a = 0, b = 0 }) => (
```